var/spack/repos/builtin/packages/httpie/package.py

```diff
@@ -26,12 +27,17 @@ class Httpie(PythonPackage):
         deprecated=True,
     )
 
+    depends_on("python", type=("build", "run"))
```

Comment:
This is already added by the base class, you don't need to add it again

Reply:
Gotcha! Removed.